src/components/layout/header/Header.tsx

```diff
@@ -164,10 +164,10 @@ const Header = (): JSX.Element => {
                             <Help />
                             <About />
                             {userRole === 'ADMIN' && (
-                                <MenuItem>
-                                    <Link href={PATHS.USERS}>
-                                        Users Management
-                                    </Link>
+                                <MenuItem
+                                    onClick={() => setLocation(PATHS.USERS)}
+                                >
+                                    Users Management
```

Comment:
During the rebase, make sure to rename this item `Users` instead of `Users Management`

Reply:
Yep no worries 😉